src/tool.yml

```
@@ -67,3 +67,55 @@ tools:
           example: /in/streams.tif
           extension: .tif
           description: .tif file for river network.
+    define_run_printouts:
+      title: Creates the printout times for the Model run 
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)